### PR TITLE
{lib}[intel/2017b] TensorFlow v1.5.0 w/ Python 3.6.3 (built from source)

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.5.0-intel-2017b-Python-3.6.3.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+patches = [
+    'TensorFlow-%(version)s_swig-env.patch',
+    'TensorFlow-%(version)s_icc-fixes.patch',
+    'TensorFlow-%(version)s_remove-msse-hardcoding.patch',
+]
+checksums = [
+    '0642781c3a3a8c2c4834b91b86aec385f0b2ada7d721571458079478cc5b29c8',  # v1.5.0.tar.gz
+    '53807290f1acb6a0f2a84f1a0ad9f917ee131c304b3e08679f3cbd335b22c7ef',  # TensorFlow-1.4.0_swig-env.patch
+    '27f88f61627bf2190e236e350a3651845e8c28de46792c1c5319681a4a4614c7',  # TensorFlow-1.4.0_icc-fixes.patch
+    '71b9d298462f819c5d223c1502305bffc9d29c50e683d04fca8912b59410a142',  # TensorFlow-1.4.0_remove-msse-hardcoding.patch
```
Comment (boegel) [inline easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.5.0-intel-2017b-Python-3.6.3.eb]: @akesandgren Comment is wrong here, patches are named `TensorFlow-1.5.0_*.patch`, just reuse `eb --inject-checksums` to fix?

File: easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.5.0-intel-2017b-Python-3.6.3.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+]
+
+builddependencies = [
+    ('Bazel', '0.7.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.5.0-intel-2017b-Python-3.6.3.eb]: Any reason you're not using `Bazel` 0.10.0 like we do for `TensorFlow-1.5.0-foss-2017b-Python-3.6.3.eb` (required easyconfig is already there)?

**Comment by boegel** [comment]
@akesandgren Ah, I forgot about this, you need a patch to use `Bazel` 0.10, see #5819... Sorry about that (although it's not really my fault ;-)).

**Comment by akesandgren** [comment]
Yeah, i saw that patch and tried to build wihtout it here and it worked so i sent it without the patch to se if it failed for you.

File: easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.5.0-intel-2017b-Python-3.6.3.eb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    'TensorFlow-%(version)s_icc-fixes.patch',
+    'TensorFlow-%(version)s_remove-msse-hardcoding.patch',
+    'TensorFlow-%(version)s_fix-Bazel-version-check.patch',
+]
+checksums = [
+    '0642781c3a3a8c2c4834b91b86aec385f0b2ada7d721571458079478cc5b29c8',  # v1.5.0.tar.gz
+    '53807290f1acb6a0f2a84f1a0ad9f917ee131c304b3e08679f3cbd335b22c7ef',  # TensorFlow-1.5.0_swig-env.patch
+    '310c5923582ca8536d3713ed8d9ac55293bfb6325ddd5f5d120e45693fd18bd2',  # TensorFlow-1.5.0_icc-fixes.patch
+    'f7d7520d7f1aa7d7d13e7ca9df7cf91b1be1d56a9d9b7821826fdc593256f7bd',  # TensorFlow-1.5.0_remove-msse-hardcoding.patch
+    '2809eb0c905961490459ab347ad78bad9a86f3c6c1c8bd461117e779ad0e4b21',  # TensorFlow-1.5.0_fix-Bazel-version-check.patch
```
Comment (boegel) [inline easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.5.0-intel-2017b-Python-3.6.3.eb]: line too long, fixed in https://github.com/akesandgren/easybuild-easyconfigs/pull/4

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2704.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/71756eaef086c93f7276f9638a27cd00 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2030.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7ea4f521e3c3fc4990f776370f9c2277 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**